plugins/fbs/fbs.js

```diff
@@ -314,7 +314,88 @@ FBS.prototype.block = function block(username, reason) {
 module.exports = function (options, imports, register) {
   var bus = imports.bus;
 
-  // Create FBS object to use in tests.
+  // Defines the configuration for the online checker below.
+  var onlineState = {
+    'online': true,
+    'threshold': 5,
+    'currentHold': 5,
```

Comment:
What is the meaning of currentHold. Perhaps another word could make med understand it :)
Perhaps
  successfulOnlineChecks